components/eamxx/src/share/tests/subfield_tests.cpp

```diff
@@ -0,0 +1,389 @@
+#include <catch2/catch.hpp>
+#include <numeric>
+
+#include "ekat/kokkos/ekat_subview_utils.hpp"
```

Comment:
Some of these includes are not needed in this test. E.g., `field_manager.hpp` or the ekat pack headers.